modules/light-clients/07-tendermint/store.go

```diff
@@ -50,6 +50,16 @@ func setClientState(clientStore storetypes.KVStore, cdc codec.BinaryCodec, clien
 	clientStore.Set(key, val)
 }
 
+func getClientState(store storetypes.KVStore, cdc codec.BinaryCodec) (*ClientState, bool) {
```

Comment:
I kind of wonder if in the long term, the sets/gets should be managed by the keeper (maybe through an internal api)

I guess in my head we have:
- light client module (handlers)
- keeper (store manager)
- types (client state, consensus state)
- msg server (probably implemented by light client module?)

The keeper, would embed the codec and storeProvider and likely be internal. The msg server would be attached to light client module? 

Currently the keeper and this function are internal, so there's room to rework them

Reply:
I will open an issue for this and add needs discussion. This can be tackled in future